utils/build.ps1

```diff
@@ -102,6 +102,9 @@ in batch file format instead of executing them.
 .PARAMETER HostArchName
 The architecture where the toolchain will execute.
 
+.PARAMETER EnableMimalloc
```

Comment:
I wonder if we should generalise this to something like `-allocator ...` to experiment with other allocators.

Reply:
Done